src/main/java/com/systelab/seed/features/patient/controller/dto/PatientResponseDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.systelab.seed.features.patient.controller.dto;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Data
+public class PatientResponseDTO {
+
+    private UUID id;
```

Comment:
Just for consistency with other DTOs, I think we should add the @Schema descriptions indicating what each field stands for.

Reply:
Done